documentation/installation/installation.tex

```diff
@@ -221,7 +229,7 @@ \subsection{Windows}
 
 export DYNAWO_LOCALE=en_GB
 export DYNAWO_RESULTS_SHOW=true
-export DYNAWO_BROWSER=xdg-open
+export DYNAWO_BROWSER=firefox
```

Comment:
the xdg-open is more general as firefox might not be installed

Reply:
agree but I found it more confusing for people that don't what it is. and i am pretty sure on every linux distrib firefox is installed as well as xdg-open.

Reply:
I don't see why we would assume what is the favorite browser of the user. It could be chrome, chromium, opera ... Ir someone doesn't know about this default value, they can either make a research on it or modify it as proposed in the doc.

Reply:
@adriguir your opinion ?

Reply:
Personnaly I didn't know about xdg-open (and has never searched for it because it was working natively on our Fedora). I also agree with Gautier on the firefox default installation on most of Linux distributions (at least classical ones).

I would thus go for Firefox but what is more important for me is that it should be clearly indicated (if the curves can't be displayed because it is not installed) where and how modifying it.

Reply:
it is already the case with 
``` bash
verify_browser() {
  if [ ! -x "$(command -v $DYNAWO_BROWSER)" ]; then
    error_exit "Specified browser DYNAWO_BROWSER=$DYNAWO_BROWSER not found."
  fi
}
```

Reply:
I will accept this decision if everyone agrees on it but I still strongly think that someone using chrome by default wouldn't like to have firefox popping each time he gets a result and potentially prompting a window to propose to become the default browser, whereas his chrome is opened on the other side.

Reply:
Let's move on with the Firefox choice and see what happens. 
I suppose that if an user is really upset by this, he/she will be able to modify it easily. And if we manage to have some one telling us that this is a problem, this will be good news because it means we have had enough people playing with it to find one to be upset with it.

Reply:
lol!